### ISSUE TITLE
Improve 'token list ended unexpectedly` error message

### ISSUE THREAD

**Issue by lloyddewit** [lloyddewit/RScript#6]
In [R-Instat issue #7199](https://github.com/africanmathsinitiative/R-Instat/issues/7199) @patowhiz states:

> For detected invalid R like in this case, should it have given a hint about the `each` parameter value missing? As a developer using the library, I would expect the class to give me a hint about the invalid R and possibly point out the exact invalid `token` I guess that may not have been a priority to implement, if so the `undefined` behavior seems fine, the comment clearly explained it.

**Error thrown** ![Rscript_error](https://user-images.githubusercontent.com/21179007/152970571-0883d901-57b9-412d-9f7d-259338310078.PNG)

**Comment by Patowhiz** [comment]
@lloyddewit thanks. Would you like me to have a look at it? I always find it interesting when I look at how you design and write your code. 
I also acknowledge it may be more time efficient if you look at it.

**Comment by lloyddewit** [comment]
@Patowhiz thanks, though there's not much interesting in this part of the code and enhancing the error message is straightforward.
'RScript' is currently only on my personal github repo, it's not set up yet for general maintenance.
In December, I also converted RScript to Python ([here](https://github.com/lloyddewit/rpyscript)) and it may be better to do future development on the Python version. It depends if/how R-Instat and OpenCDMS use the library.
In summary, I'm happy just to collect known issues here. It's not a priority to fix this issue now.

**Comment by Patowhiz** [comment]
@lloyddewit sounds good. Thanks.
